inspirehep/config.py

```diff
@@ -82,7 +85,7 @@
 
 # Database
 # ========
-SQLALCHEMY_DATABASE_URI = "postgresql+psycopg2://inspirehep:dbpass123@localhost:5432/inspirehep"
+SQLALCHEMY_DATABASE_URI = "postgresql+psycopg2://inspirehep:inspirehep@localhost:5432/inspirehep"
```

Comment:
I think that you should also change password in services.yml for db service if you want to change it correctly.

Reply:
my bad!